src/test/common/test_crc32c.cc

```diff
@@ -265,3 +265,28 @@ TEST(Crc32c, RangeNull) {
     ASSERT_EQ(crc, *check);
   }
 }
+
+TEST(Crc32c, zeros_performance) {
+  utime_t start;
+  utime_t end;
+
+  for (size_t scale=1; scale < 31; scale++)
+  {
+    size_t size = (1<<scale) + rand()%(1<<scale);
+    start = ceph_clock_now();
+    uint32_t crc_a = ceph_crc32c(111, nullptr, size);
+    end = ceph_clock_now();
+    std::cout << "regular  method. size=" << size << " time= " << (double)(end-start)
```

Comment:
@aclamk Both implementation perform one call in sub-milisecond time on any reasonable CPU, with a lot of things getting in the way *and* additionally you don't use high-resolution clock. All that will give you huge statistical error making such measurements pointless. Running each function in a separate loop and measuring either entire loop time, or dividing loop time by iteration count will give you more reliable, repeatable results.